src/service/poc_mobile.proto

```diff
@@ -1077,6 +1080,9 @@ message subscriber_mapping_activity_req_v1 {
   bytes carrier_pub_key = 7;
   /// Signed payload of the mapping activity request
   bytes signature = 8;
+  // String representing the entity key
+  // Overrides the subscriber_id as the entity_key to send mapping rewards to
+  string entity_key = 9;
```

Comment:
can we make the name more descriptive? might be to verbose but something like `reward_override_entity_key`??  I know the comment explains the purpose, but the field just being named `entity_key` won't help when reading the code that utilizes it

Reply:
I like it +1